articles/tokens/refresh-token/current/index.md

```diff
@@ -13,36 +13,39 @@ useCase:
 
-1. By enabling the **OIDC Conformant** flag for an Application
-2. By passing an `audience` to the `/authorize` endpoint
+1. Enabling the **OIDC Conformant** flag for an Application
+2. Passing an `audience` to the `/authorize` endpoint of the Authentication API 
 
-For more information on our authentication pipeline, refer to [Introducing OIDC Conformant Authentication](/api-auth/intro).
+For more information on our authentication pipeline, see [Introducing OIDC-Conformant Authentication](/api-auth/intro).
 :::
 
 ## Overview
 
-The response of an [authentication request](/api-auth) can result in an Access Token and/or an ID Token being issued by Auth0. The  Access Token is used to make authenticated calls to a secured API, while the ID Token contains user profile attributes represented in the form of _claims_. Both JWTs have an expiration date indicated by the `exp` claim (among other security measures, like signing).
+Auth0 can issue an Access Token and/or an ID Token in response to an [authentication request](/api-auth). You can use Access Tokens to make authenticated calls to a secured API, while the ID Token contains user profile attributes represented in the form of *claims*. Both are [JWTs](/jwt) and therefore have expiration dates indicated using the `exp` claim, as well as security measures, like signatures.
 
-A Refresh Token allows the application to request Auth0 to issue a new Access Token or ID Token directly, without having to re-authenticate the user. This will work as long as the Refresh Token has not been revoked.
+A Refresh Token allows the application to ask Auth0 to issue a new Access Token or ID Token without having to re-authenticate the user. This will work as long as the Refresh Token has not been revoked.
 
-## Restrictions
+## Restrictions on Refresh Token Usage
 
 You can only get a Refresh Token if you are implementing the [Regular Web App Login Flow](/flows/concepts/regular-web-app-login-flow), [Native/Mobile Login Flow](/flows/concepts/mobile-login-flow), or [Resource Owner Password Grant](/api-auth/grant/password).
 
-A Single-Page Application (normally implementing [Single-Page Login Flow](/flows/concepts/single-page-login-flow)) should not under any circumstances get a Refresh Token. The reason for that is the sensitivity of this piece of information. You can think of it as user credentials, since a Refresh Token allows a user to remain authenticated essentially forever. Therefore you cannot have this information in a browser, it must be stored securely.
+A Single-Page Application (normally implementing [Single-Page Login Flow](/flows/concepts/single-page-login-flow)) should not, under any circumstances, get a Refresh Token. The reason for that is the sensitivity of this piece of information. You can think of the token as a user credential since a Refresh Token allows a user to remain authenticated essentially forever. Therefore you cannot have this information in a browser; it must be stored securely.
```

Comment:
This sounds odd to me (maybe it's just me). I'd maybe go for something like: 

```suggestion
A Single-Page Application (normally implementing [Single-Page Login Flow](/flows/concepts/single-page-login-flow)) should not ever receive a Refresh Token. A Refresh Token is essentially a user credential which allows a user to remain authenticated indefinitely. This sensitive information should be stored securely and not exposed client-side in a browser.
```

(and then maybe a link to something about storing tokens? :smiling-imp:)